health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/UserUpload.js

```
@@ -366,6 +366,57 @@ const UserUpload = React.memo(() => {
+            action: "create",
+            campaignId: id,
+            additionalDetails: {
+              source: "microplan"
+            }
+          }
+        }
+      }
+      try {
+        await axios.post(reqCriteria.url, reqCriteria.body);
+      } catch (error) {
+        var errorLabel;
```

Biome

[error] Use let or const instead of var.

A variable declared with var is accessible in the whole body of the function. Thus, the variable can be accessed before its initialization and outside the block where it is declared.
See MDN web docs for more details.
Unsafe fix: Use 'let' instead.


(lint/style/noVar)